CMakeLists.txt

```diff
@@ -164,6 +164,22 @@ if (BUILD_OPENSSL AND OE_TRUSTZONE)
   message(FATAL_ERROR "BUILD_OPENSSL is not supported on ARM yet.")
 endif ()
 
+set(DEFAULT_TEST_ENCLAVE_CRYPTO_LIB
+    "MbedTLS"
```

Comment:
It is better to 
- use  lower case characters
- perform case-insensitive compare

The mbed tls website (https://tls.mbed.org/about-us) itself does not follow a consistent capitalization scheme for the library,. It is not really obvious that `MbedTLS` is the preferred way to spell the option.

Reply:
Done.

Reply:
Actually in prose it's always "Mbed TLS" with a space in it.  https://www.trustedfirmware.org/projects/mbed-tls/ is the official home (like OE is under CCC) and https://github.com/ARMmbed/mbedtls is the github project.  Both are consistent. (The tls.mbed.org site is obsolete and has the notice on it "The information on this website will be retained while we migrate but please note it may be out of date.")